codespeed/models.py

```diff
@@ -73,6 +76,31 @@ def save(self, *args, **kwargs):
         super(Project, self).save(*args, **kwargs)
 
 
+class HistoricalValue(object):
+    def __init__(self, name=None, val=0, color='none'):
+        self.name = name
+        self.val = val
+        self.color = color
+
+    def if_significant_update_to(self, val, color, name):
```

Comment:
I find the name not easy to read. How about "update_if_significant_change" or "update_to_biggest_change"?